lab-shawn/lib/greet.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports.greet = function(name){
+  if(arguments.length === 0) throw new Error('No Name Provided');
+  console.log(`Hello, ${name}`) ;
+  return (`Hello, ${name}`) ;
+};
```

Comment:
I noticed you're invoking this method in index.js but you are passing it a different argument that this function isn't accounting for. In index.js you are passing process.argv[2] to this method but assign process.argv[2] to a different variable 'arg'. This function is expecting a variable called name.